mgmt/BaseManager.h

```diff
@@ -83,27 +83,27 @@
  ***********************************************************************/
 
 // Signal flows: traffic server -> traffic manager
-#define MGMT_SIGNAL_PID 0
-#define MGMT_SIGNAL_MACHINE_UP 1 /* Data is ip addr */
-#define MGMT_SIGNAL_MACHINE_DOWN 2
-#define MGMT_SIGNAL_CONFIG_ERROR 3 /* Data is descriptive string */
-#define MGMT_SIGNAL_SYSTEM_ERROR 4
-#define MGMT_SIGNAL_LOG_SPACE_CRISIS 5
-#define MGMT_SIGNAL_CONFIG_FILE_READ 6
-#define MGMT_SIGNAL_CACHE_ERROR 7
-#define MGMT_SIGNAL_CACHE_WARNING 8
-#define MGMT_SIGNAL_LOGGING_ERROR 9
-#define MGMT_SIGNAL_LOGGING_WARNING 10
+#define MGMT_SIGNAL_PID 100
```

Comment:
These need to be unified, it's a bad idea to have effectively the same values in different headers.

Reply:
I think there supposed to be the exact same values. There's a comment "copy from mgmt/BaseManager.h"